### ISSUE TITLE
The code doesn't include validate function, is it?

### ISSUE THREAD

**Issue by YeeHoran** [WuJie1010/Facial-Expression-Recognition.Pytorch#128]
Dear,

I see through the FER code "main_FER.py" carefully, but don't see any code about validation. However, there exist public test and private test. I wonder does the public test means to replace validation?

Thank you!

Yee Horan

**Comment by WuJie1010** [comment]
FER doesn't have validation set. It only has public test and private test.

**Comment by YeeHoran** [comment]
oh, thank you for your reply!
